src/Uno.UWP/Devices/Midi/MidiOutPort.iOSmacOS.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.Devices.Enumeration.Internal;
+using Uno.Devices.Enumeration.Internal.Providers.Midi;
+using Windows.Devices.Enumeration;
+using Windows.Foundation;
+using CoreMidi;
+using MidiInfo = CoreMidi.Midi;
+using System.Net;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Storage.Streams;
+
+namespace Windows.Devices.Midi
+{
+	public partial class MidiOutPort : IDisposable
+	{
+		private MidiEndpoint _endpoint;
+		private MidiClient _client;
+		private MidiPort _port;
+
+		private MidiOutPort(MidiEndpoint endpoint)
+		{
+			_endpoint = endpoint;
+			_client = new MidiClient(Guid.NewGuid().ToString());
+		}
+
+		public string DeviceId { get; private set; }
+
+		internal async Task OpenAsync()
+		{
+			var completionSource = new TaskCompletionSource<MidiDevice>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'completionSource'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)